modules/flowable-engine/src/main/java/org/flowable/engine/impl/persistence/entity/ExecutionEntityImpl.java

```diff
@@ -168,14 +168,19 @@ public class ExecutionEntityImpl extends AbstractBpmnEngineVariableScopeEntity i
      */
     protected Integer processDefinitionVersion;
 
+    /**
+     * persisted reference to the process definition version.
```

Comment:
```suggestion
     * Persisted reference to the process definition version.
```
I know this is insignificant but to be consistent with other Javadoc (see the next couple of variables)